### DISCUSSION TITLE
NuGet's Garnet and .NET Framework

### DISCUSSION THREAD

**Discussion by dxluck** [microsoft/garnet#487] [Q&A] [answered]
Will the NuGet version of "Microsoft.Garnet 1.0.14" work with the .NET Framework?
I tried to install the NuGet version of "Microsoft.Garnet 1.0.14" into a .NET Framework 4.8 project using Visual Studio 2022, but the installation failed. At the end of the installation, I get the following message:

> Could not install package 'Microsoft.Garnet 1.0.14'. You are trying to install this package into a project that targets '.NETFramework,Version=v4.8', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.

Are the system requirements listed somewhere?

**Comment by badrishc** [comment, marked as answer]
Did the earlier versions of Garnet work with .NET Framework? We have not supported it since the start.

**Reply by dxluck** [reply]
Thanks for your answer. I guess I was mistaken. Nowadays, when people simply say ".NET", it doesn't usually include ".NET Framework".
